app/src/main/cpp/skyline/nce/instructions.h

```diff
@@ -19,13 +19,8 @@ namespace skyline::nce {
         struct Brk {
             /**
              * @brief Creates a BRK instruction with a specific immediate value, used for generating BRK opcodes
-             * @param value The immediate value of the instruction
              */
-            constexpr Brk(u16 value) {
-                sig0 = 0x0;
-                this->value = value;
-                sig1 = 0x6A1;
-            }
+            constexpr Brk(u16 value) : sig0(0x0), value(value), sig1(0x6A1) {}
```

Comment:
It's very neat that C++20 finally added default initialization for bitfields, this file looks significantly better with it.